### PR TITLE
I add files to persist contact data of the form

### PR DESCRIPTION
**Pull request by DanteIturri**
I add necessary files to capture contact data and method to persist in database
Files Add: 
ContacEntity, Contact.dto, ContactMapper, ContactRepository, ContactService

### CODE REVIEW

File: src/main/java/com/alkemy/ong/entity/ContactEntity.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "contact")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE contact SET deletedAt = true WHERE id = ?")
```
Comment (JonathanBolatti) [inline src/main/java/com/alkemy/ong/entity/ContactEntity.java:19]: Shouldn't the SQL statement and clause be soft_delete?

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/entity/ContactEntity.java:19]: no the ticket asks me to name it like that and to use it as softdelete

Comment (anavduo) [reply src/main/java/com/alkemy/ong/entity/ContactEntity.java:19]: That's right I founded weird but has a different name. 
I think the table name should be in plural "contacts"

File: src/main/java/com/alkemy/ong/mapper/ContactMapper.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.alkemy.ong.mapper;
+
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/mapper/ContactMapper.java:2]: I believe this is part of my ticket to solve the endpoint, yours mentions Entity, Repository and Service only

File: src/main/java/com/alkemy/ong/service/ContactService.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ContactService {
+    @Autowired
+    ContactMapper contactMapper;
+
+    @Autowired
+    ContactRepository contactRepository;
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/service/ContactService.java:16]: This @Autowired bean means we are no implementing the mapstruct dependencia. My suggestion please let me write the mapper so we are consistent in the implementation of the mapstruct

File: src/main/java/com/alkemy/ong/dto/ContactDto.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package com.alkemy.ong.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/dto/ContactDto.java:4]: this also I believe I need to write it

File: src/main/java/com/alkemy/ong/entity/ContactEntity.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+@Where(clause = "deletedAt = false ")
+public class ContactEntity {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name= "uuid", strategy = "uuid2")
+    @Column(name = "id", nullable = false)
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/entity/ContactEntity.java:26]: In the previous entities we have name all the tables in lower case, just to be consistents

**Comment by anavduo** [comment]
Good job!

**Pull request merged**